src/state/zustand/swapperStore/utils.ts

```diff
@@ -18,6 +21,30 @@ type SumProtocolFeesToDenomArgs = {
   protocolFees: Record<AssetId, ProtocolFee>
 }
 
+/**
+ * Subtracts basis point amount from a given value.
+ *
+ * @param value The value to subtract basis points from.
+ * @param basisPoints The number of basis points to subtract.
+ * @param roundUp Round up the result to the nearest integer.
+ * @returns The new number that is the input value minus the basis points of the value.
+ */
+export const subtractBasisPointAmount = (
+  value: string,
+  basisPoints: string,
+  roundUp?: boolean,
+): string => {
+  const bigNumValue = bn(value)
+
+  // Basis point is 1/100th of a percent
+  const percentValue = convertBasisPointsToDecimalPercentage(basisPoints)
+  const subtractValue = bigNumValue.times(percentValue)
+
+  // Subtract basis points from the original value
+  const resultValue = bigNumValue.minus(subtractValue)
+  return roundUp ? resultValue.toFixed(0, 0) : resultValue.toFixed()
```

Comment:
```suggestion
  return roundUp ? resultValue.toFixed(0, BigNumber.ROUND_UP) : resultValue.toFixed()
```

Reply:
Absolutely this! https://github.com/shapeshift/web/pull/4568/commits/00a4e3791de782011bebb7ba2f7b46a5b5328f51